tests/test_bears/TestBear.py

```diff
@@ -5,6 +5,9 @@ class TestBear(LocalBear):
 
     def run(self, file, filename, result=False, exception: bool = False):
         if result is True:
+            if file:
+                for line in file.split('\n'):
```

Comment:
Errr ... in the bears, the run method is usually `run(filename, file,...)` and I believe `file` is typically an array of lines.